lib/sproxyd.js

```diff
@@ -287,79 +287,76 @@ class SproxydClient {
         const host = this.getCurrentBootstrap();
         const isBatchDelete = key === '.batch_delete';
         const newKey = key || keygen(this.cos, params);
-        const req = this._createRequestHeader(method, headers, newKey, params,
-            log);
+        const req = this._createRequestHeader(method, headers, newKey, params, log);
+        const logInfo = {
```

Comment:
Instead of using a common object, you should be able to use `log.addDefaultFields({...})` when creating the request logger.